userguide/jails.rst

```diff
@@ -1413,8 +1365,8 @@ using :command:`iocage`.
 :command:`iocage` has several options to help users:
 
 * There is built-in help displayed by entering
-  :samp:`iocage --help | less`. Each subcommand also has help,
-  displayed by giving the subcommand name followed by the
+  :samp:`iocage --help | less`. Each subcommand also has help.
+  Display them by giving the subcommand name followed by the
```

Comment:
Not "them".

Display help by adding the :literal:`--help` flag after the subcommand name.